### DISCUSSION TITLE
Google Keep

### DISCUSSION THREAD

**Discussion by jirigary** [zepp-health/zeppos-forum#356] [General]
Hi guys, is there anyone capable of developing app for Zepp OS similar to Google Keep with connection with Google Keep?

**Comment by 3DTreeDee** [comment]
This one is interesting, since I also use keep on daily basis and will be ideal to have a checklist while doing shopping. I'll start checking if it's feasible once I get my device, I'll ping back my findings once I know if it's a go to start development
